### ISSUE TITLE
Add link button to attached server profiles

### ISSUE THREAD

**Issue by jhg03a** [apache/trafficcontrol#2784]
There isn't presently a way to mass-edit what servers are attached to a server profile.

I'd like to see a link button like <base_url>/#!/profiles/880/parameters on <base_url>/#!/profiles/880/servers

**Comment by mitchell852** [comment]
> There isn't presently a way to mass-edit what servers are attached to a server profile.

@jhg03a is this the functionality you are looking for? if so, i'm confused what the 2nd sentence is about.

**Comment by jhg03a** [comment]
Correct.  The second sentence is an example of what I was looking for.

**Comment by mitchell852** [comment]
a better UI to manage server profiles/params is coming with TPv2. stay tuned!
